src/DotNetWorker/Context/Features/DefaultModelBindingFeature.cs

```diff
@@ -48,8 +52,23 @@ public DefaultModelBindingFeature(IEnumerable<IConverter> converters)
                 if (TryConvert(converterContext, out object? target))
                 {
                     _parameterValues[i] = target;
-                    continue;
                 }
+                else if (source is not null)
+                {
+                    // Don't initialize this list unless we have to
+                    if (errors is null)
+                    {
+                        errors = new List<string>();
+                    }
+
+                    errors.Add($"Cannot convert input parameter '{param.Name}' to type '{param.Type.FullName}' from type '{source.GetType().FullName}'.");
+                }
+            }
+            
+            // found errors
+            if (errors is not null)
+            {
+                throw new FunctionInputConverterException($"Error converting {errors.Count} input parameters for Function '{context.FunctionDefinition.Name}': {string.Join(" ", errors)}");
```

Comment:
nit: should the join be with `Environment.NewLine` so we're providing each error in a new line?